StaticSiteGenerators(tables).rst

```diff
@@ -14,42 +14,40 @@ Starting notes:
 * What are static site generators?
 * From the June WTD Newletter:
 
-WHAT IS A STATIC SITE GENERATOR
-===============================
+WHAT IS A STATIC SITE GENERATOR?
+================================
 
-A static site generator is a tool that can publish a collection of markup files to a site. Popular static site generators include Sphinx, Jekyll, and Asciidoctor.
+A static site generator (SSD) is a tool that can convert a collection of markup files into web pages. Most SSDs include programs that host the resulting web site, but because the output doesn't depend on a specific hosting server, you have many options for deploying the content. Popular static site generators include Sphinx, Jekyll, and Asciidoctor.
```

Comment:
Why are you using SSD instead of SSG here?

Reply:
I... don't know. What a strange brain fart. I'll fix that.